actionview/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   `translate` now allows `default: [[]]` again for a default value of `[]`.
+
+    Fixes a regression introduced in 4.2.1.
+
+    See #19640 and the fix in #19649.
```

Comment:
For future reference, note I [edited this a bit](https://github.com/rails/rails/commit/07875e2aa6ddf4c37b5997d6834bcb27ea50d02c#diff-d5f0229b1e83113a6aedf860e32398f9): PRs don't reference themselves, and 4.2.1 doesn't exist in master's history.